src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalSuggestTelemetry.ts

```diff
@@ -33,38 +33,38 @@ export class TerminalSuggestTelemetry extends Disposable {
 			return;
 		}
 		this._acceptedCompletions = this._acceptedCompletions || [];
-		this._acceptedCompletions.push({ label: typeof completion.label === 'string' ? completion.label : completion.label.label, kindLabel: completion.kindLabel });
+		this._acceptedCompletions.push({ label: typeof completion.label === 'string' ? completion.label : completion.label.label, type: completion.type });
 	}
 	private _sendTelemetryInfo(fromInterrupt?: boolean, exitCode?: number): void {
 		const commandLine = this._promptInputModel?.value;
 		for (const completion of this._acceptedCompletions || []) {
 			const label = completion?.label;
-			const kind = completion?.kindLabel;
-			if (label === undefined || commandLine === undefined || kind === undefined) {
+			const type = completion?.type;
+			if (label === undefined || commandLine === undefined || type === undefined) {
 				return;
 			}
 
-			let outcome: CompletionOutcome;
+			let outcome: string;
 			if (fromInterrupt) {
-				outcome = CompletionOutcome.Interrupted;
+				outcome = 'Interrupted';
 			} else if (commandLine.trim() && commandLine.includes(label)) {
-				outcome = CompletionOutcome.Accepted;
+				outcome = 'Accepted';
 			} else if (inputContainsFirstHalfOfLabel(commandLine, label)) {
-				outcome = CompletionOutcome.AcceptedWithEdit;
+				outcome = 'AcceptedWithEdit';
 			} else {
-				outcome = CompletionOutcome.Deleted;
+				outcome = 'Deleted';
 			}
 			this._telemetryService.publicLog2<{
-				kind: string | undefined;
-				outcome: CompletionOutcome;
```

Comment:
A const enum should work fine, I wouldn't change to using strings when filling in the outcome. It's possible the `outcome: CompletionOutcome;` is throwing it off and only that is needed?